src/artemis/stores/mapgraph/selections.cljs

```diff
@@ -9,7 +9,7 @@
 (defn has-args? ^boolean [selection]
   (boolean (:arguments selection)))
 (defn type-cond? ^boolean [selection]
-  (boolean (some-> selection :selection-set first :type-condition)))
+  (boolean (some :type-condition (:selection-set selection))))
```

Comment:
instead of only checking if the first field in a selection set is a union fragment, we check to see if any of the fields in a selection set is one.